.tina/config.ts

```diff
@@ -5,6 +5,22 @@ const branch =
 
 const tokenForEveryBranch = "351c04276ed1059a64f12ee7ade4c2fab8e11562";
 
+const isValidVocaroURL = (link: string | undefined): boolean => {
+  if (!link) {
+    return true;
+  }
+
+  try {
+    const newUrl = new URL(link);
+    return (
```

Comment:
```suggestion

    return (
```
More readable with a break line before it